examples/reference-entity-model/entities/Address.entity.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
We don't need nor want separate entity files. The approach for 5.3.0 is that each model file has one "primary" entity whose name is the same as info.title. Each other entity definition is considered a structured type.

Is there a particular feature that is requiring Address and Zip to be duplicated in separate files?

Reply:
Mapping doesnt seem to work without them and complains about missing target entitiy.

Reply:
We can get them updated later to include merge matches. This is just so the nightly instance will have something for the team to build their demo and can be enhanced later.

Reply:
Hmm - that's a concern, I'll try it out locally.

Reply:
I verified that the mapping tool in HC is using "Customer" instead of the full entityTypeId. The backend does not yet support mappings that use just entityName and then try to refer to other entity definitions. I'll open a bug for this, as we do not want to go down the path of duplicating entity types again in HC.